elf/elf.go

```diff
@@ -293,31 +286,69 @@ func elfReadVersion(file *elf.File) (uint32, error) {
 	return 0, nil
 }
 
-func prepareBPFFS(namespace, name string) (string, error) {
+func createPinPath(path string) (string, error) {
 	err := bpffs.Mount()
```

Comment:
Can you put this on one line: `if err := bpffs.Mount(); err != nil {` Same of the next error check below.